pkg/kubectl/describe.go

```diff
@@ -1801,7 +1833,7 @@ func describeServiceAccount(serviceAccount *api.ServiceAccount, tokens []api.Sec
 			tokenSecretNames = []string{}
 		)
 
-		for _, s := range serviceAccount.ImagePullSecrets {
+		for _, s := range saPullSecrets {
```

Comment:
for example, if you ran this command with a user that doesn't have permission to list secrets, you would see no pull secrets in the describe, even if the service account referenced some